cypress/integration/customizer/general/buttons.spec.js

```diff
@@ -0,0 +1,115 @@
+let colors = {
```

Comment:
colors could be a const instead of let since the variable does not receive a new value (if opting for not using it stored into a fixture file)